src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java

```diff
@@ -2321,6 +2321,14 @@ public void setBrowserSizeToMaximum() {
         getSeleniumHelper().setWindowSizeToMaximum();
     }
 
+    public void setBrowserPositionXY(int xPos, int yPos) {
+		getSeleniumHelper().setBrowserPositionXY(xPos, yPos);
+	}
+
+    public void setBrowserPositionToHome() {
```

Comment:
Please remove this method. If a user wants that they should just make a scenario for it.